migrations/1588865214157-addEnableStudentTeamCreation.js

```diff
@@ -0,0 +1,45 @@
+const { info, error } = require('../src/logger');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['error' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5488274)